apps/css-workshop/src/components/Button.astro

```diff
@@ -11,8 +11,13 @@ const { class: className, variant, size, active, shift, iuiType, ...props } = As
 ---
 
 <button
-  class:list={['iui-button', { 'iui-button-dropdown': iuiType === 'dropdown' }, className]}
-  data-iui-variant={iuiType === 'feedback' ? 'idea' : variant}
+  class:list={[
+    'iui-field iui-button',
```

Comment:
We also need `iui-button-base` within this component, to match React implementation. Maybe can be handled in #2012

Reply:
cc @Ben-Pusey-Bentley

Reply:
Okay, will do.